backend/src/impl/internal_models/system_model.py

```diff
@@ -235,5 +234,53 @@ def update_analysis_cases():
             generate_system_update_values(),
             session=session,
         )
-        output_collection = DBUtils.get_system_output_collection(self.system_id)
-        DBUtils.insert_many(output_collection, update_analysis_cases(), False, session)
+
+    def get_system_output(self, output_ids: list[int] | None) -> list[dict]:
```

Comment:
Should this be `get_system_outputs` plural, because it's getting multiple ones?

Additionally, maybe it should be `get_raw_system_output` or `get_system_output_dict` because `system_output_from_dict` has not been called yet?

Also, it'd be nice to have a docstring.

Reply:
Thanks for the comment! I renamed `get_system_ouput` -> `get_raw_system_outputs` and `get_analysis_case` to `get_raw_analysis_cases`. I also added docstrings for both of them.